### ISSUE TITLE
There was a problem when opening the new image

### ISSUE THREAD

**Issue by jtojnar** [lmanul/gimp-texturize#8]
When I try to execute _Texturize_ in GIMP 2.10.28 (default values in the dialogue), nothing happens. The following message appears in the status bar: There was a problem when opening the new image.

**Comment by jtojnar** [comment]
Maybe it is just new and unsaved images, saved XCF files and PNG files. It seems to work with most of the JPEGs I tried.

**Comment by lmanul** [comment]
Hmm, I've tried with a PNG image but it seems to work fine for me. Here is what I tried:

* Download an example image: `wget "https://lmanul.github.io/gimp-texturize/Images/Strawberries_base.jpg"`
* Convert it to PNG with `convert`
* Open the PNG image in GIMP
* Open the Texturize plugin
* Accept the defaults

**Comment by jtojnar** [comment]
Yeah, that one works for me. https://www.golem.de/staticrl/images/golem-rss.png, for example, does not.
